src/main/java/slimeknights/tconstruct/smeltery/TinkerSmeltery.java

```diff
@@ -112,7 +120,7 @@ static void registerBlockLists(final FMLCommonSetupEvent event) {
     // smeltery adds in tank, glass and drains
     builder = ImmutableSet.builder();
     builder.addAll(searedBlocks);
-    //builder.add(searedTank);
+    //builder.add(TinkerSmeltery.searedTank.values());
```

Comment:
Why is this still commented out? Should be added to the set.

Reply:
It doesn't like it.
```log
/home/hanetzer/MinecraftMods/TinkersConstruct/src/main/java/slimeknights/tconstruct/smeltery/TinkerSmeltery.java:123: error: no suitable method found for add(List<SearedTankBlock>)
    builder.add(TinkerSmeltery.searedTank.values());
           ^
    method com.google.common.collect.ImmutableCollection.Builder.add(Block) is not applicable
      (argument mismatch; List<SearedTankBlock> cannot be converted to Block)
    method com.google.common.collect.ImmutableCollection.Builder.add(Block...) is not applicable
      (varargs mismatch; List<SearedTankBlock> cannot be converted to Block)
    method ArrayBasedBuilder.add(Block) is not applicable
      (argument mismatch; List<SearedTankBlock> cannot be converted to Block)
    method ArrayBasedBuilder.add(Block...) is not applicable
      (varargs mismatch; List<SearedTankBlock> cannot be converted to Block)
    method com.google.common.collect.ImmutableSet.Builder.add(Block) is not applicable
      (argument mismatch; List<SearedTankBlock> cannot be converted to Block)
    method com.google.common.collect.ImmutableSet.Builder.add(Block...) is not applicable
      (varargs mismatch; List<SearedTankBlock> cannot be converted to Block)
```

Reply:
Add should be addAll.

Reply:
Ahhhh.